themes/janeswalk/js/components/pages/Dashboard/DashboardResources.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import DashboardStore from './DashboardStore';
+
+//TODO: Add styling, reference styling below, also City and Country Data is not available (assumed Toronto, Canada)
+
+const DashboardResources = () => {
+  const {cityOrganizers, videoTips, files, featuredWalks} = DashboardStore.getResources();
```

Comment:
For stateless components, it's generally cleaner and more predictable to pass these in as props instead. Otherwise, you're essentially giving a stateless component a state, when that should always be managed by the parent.